text/0119-standard-system-metrics.md

```diff
@@ -0,0 +1,147 @@
+|                            |       |               |          |direction|read, write |
+|system.disk.time            |seconds|SumObserver    |Double    |device   |(identifier)|
+|                            |       |               |          |direction|read, write |
+
+#### `system.filesystem.`
+
+**Description:** System level filesystem metrics.
+|Name                         |Units|Instrument Type  |Value Type|Label Key|Label Values        |
+|-----------------------------|-----|-----------------|----------|---------|--------------------|
+|system.filesystem.usage      |bytes|UpDownSumObserver|Int64     |device   |(identifier)        |
+|                             |     |                 |          |state    |used, free, reserved|
+|system.filesystem.utilization|1    |UpDownSumObserver|Double    |device   |(identifier)        |
+|                             |     |                 |          |state    |used, free, reserved|
+
+#### `system.net.`
+
+**Description:** System level network metrics.
+|Name                       |Units      |Instrument Type  |Value Type|Label Key|Label Values                                                                                  |
+|---------------------------|-----------|-----------------|----------|---------|----------------------------------------------------------------------------------------------|
+|system.net.dropped\_packets|packets    |SumObserver      |Int64     |device   |(identifier)                                                                                  |
+|                           |           |                 |          |direction|transmit, receive                                                                             |
+|system.net.packets         |packets    |SumObserver      |Int64     |device   |(identifier)                                                                                  |
+|                           |           |                 |          |direction|transmit, receive                                                                             |
+|system.net.errors          |errors     |SumObserver      |Int64     |device   |(identifier)                                                                                  |
+|                           |           |                 |          |direction|transmit, receive                                                                             |
+|system<!--notlink-->.net.io|bytes      |SumObserver      |Int64     |device   |(identifier)                                                                                  |
+|                           |           |                 |          |direction|transmit, receive                                                                             |
+|system.net.connections     |connections|UpDownSumObserver|Int64     |device   |(identifier)                                                                                  |
```

Comment:
Not sure if it was discussed before. It seems a bit inconsistent that we use a short version `net` instead of `network` here, but full names like `memory` and `filesystem` are used for other metrics. Also I see the `network` already used in Collector https://github.com/open-telemetry/opentelemetry-collector/blob/master/receiver/hostmetricsreceiver/hostmetrics_receiver_test.go#L54-L58. Do we need to update it here or in the collector? /cc @james-bebbington

Reply:
@dmitryax thanks, changing it to "network" for consistency.